src/coord/src/catalog/migrate.rs

```diff
@@ -143,52 +140,31 @@ fn ast_rewrite_kafka_protobuf_source_text_to_compiled_0_9_13(
     stmt: &mut sql::ast::Statement<Raw>,
 ) -> Result<(), anyhow::Error> {
     fn compile_proto(schema: &str) -> Result<CsrSeedCompiledEncoding, anyhow::Error> {
-        let temp_schema_name: String = Uuid::new_v4().to_simple().to_string();
-        let include_dir = tempfile::tempdir()?;
-        let schema_path = include_dir.path().join(&temp_schema_name);
-        let schema_bytes = strconv::parse_bytes(schema)?;
-        fs::write(&schema_path, &schema_bytes)?;
-
-        match Protoc::new()
-            .include(include_dir.path())
-            .input(schema_path)
-            .parse()
-        {
-            Ok(fds) => {
-                let message_name = fds
-                    .file
-                    .iter()
-                    .find(|f| f.get_name() == temp_schema_name)
-                    .map(|file| file.message_type.first())
-                    .flatten()
-                    .map(|message| format!(".{}", message.get_name()))
-                    .ok_or_else(|| anyhow!("unable to compile temporary schema"))?;
-                let mut schema = String::new();
-                strconv::format_bytes(&mut schema, &fds.write_to_bytes()?);
-                Ok(CsrSeedCompiledEncoding {
-                    schema,
-                    message_name,
-                })
-            }
-            Err(e) => {
-                lazy_static! {
-                    static ref MISSING_IMPORT_ERROR: Regex = Regex::new(
-                        r#"protobuf path \\"(?P<reference>.*)\\" is not found in import path"#
-                    )
-                    .unwrap();
-                }
+        // Compile .proto files into a file descriptor set.
+        let path = Path::new("migration.proto");
```

Comment:
@cjubb39, to answer your question from before: it didn't seem like using a UUID here mattered, since there can only ever be one .proto file that we're compiling and so it can't possibly conflict with the name of another .proto file.